modules/configuring-layer-three-routed-topology.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-networks.adoc
+
+:_content-type: CONCEPT
+[id="configuring-layer-three-routed-topology_{context}"]
+= Configuring a routed topology
+
+The routed (_layer 3_) topology networks are a simplified topology for the cluster default network without egress. In this topology, there is a logical switch per node, each with a different subnet, and a router interconnecting all logical switches.
```

Comment:
hm, not only egress. Ingress is also not possible to these networks; they are a disconnected overlay, only allowing east/west communication, as you mention in the `note` below.